engine/graphics/proto/graphics/graphics_ddf.proto

```diff
@@ -209,20 +209,26 @@ message TextureImage
         required uint32 original_height           = 4;
         required TextureFormat format             = 5;
         repeated uint32 mip_map_offset            = 6;
-        repeated uint32 mip_map_size              = 7;    // always uncompressed (native) size
-        required bytes data                       = 8;
-        optional CompressionType compression_type = 9 [default = COMPRESSION_TYPE_DEFAULT];
-        optional uint64 compression_flags         = 10;
-        repeated uint32 mip_map_size_compressed   = 11;
-        repeated uint32 mip_map_dimensions        = 12; // w0, h0, w1, h1, ...
+        repeated uint32 mip_map_size              = 7; // always uncompressed (native) size
+        optional CompressionType compression_type = 8 [default = COMPRESSION_TYPE_DEFAULT];
+        repeated uint32 mip_map_size_compressed   = 9;
+        repeated uint32 mip_map_dimensions        = 10; // w0, h0, w1, h1, ...
+        optional uint32 data_size                 = 11;
```

Comment:
What does this represent? isn't it either `mip_map_size` or `mip_map_size_compressed`, depending on the `compression_type`?

Perhaps add a comment in the file?